src/vcpkg/sourceparagraph.cpp

```diff
@@ -78,20 +78,20 @@ namespace
                                  "parenthesized expressions).");
     DECLARE_AND_REGISTER_MESSAGE(LicenseExpressionExpectCompoundOrWithFoundWord,
                                  (msg::value),
-                                 "",
+                                 "Example of {value} is 'MIT'",
```

Comment:
```suggestion
                                 "Example of {value} is a license name, like 'MIT'",
```

I don't think the average translator knows that "MIT" is a license.

Reply:
they're translating the following sentence: `"Expected either AND or OR, found a license or exception name {value}"`, so I think it's fine not to say that here.

Reply:
Ah, makes sense. I guess there's not much context for code reviewers either XD